qiskit_experiments/framework/composite/composite_analysis.py

```diff
@@ -13,18 +13,15 @@
 Composite Experiment Analysis class.
 """
 
-from qiskit.exceptions import QiskitError
-from qiskit_experiments.framework import BaseAnalysis
-from .composite_experiment_data import CompositeExperimentData
+from qiskit.result import marginal_counts
+from qiskit_experiments.framework import BaseAnalysis, ExperimentData
 
 
 class CompositeAnalysis(BaseAnalysis):
     """Analysis class for CompositeExperiment"""
 
-    __experiment_data__ = CompositeExperimentData
-
     # pylint: disable = arguments-differ
-    def _run_analysis(self, experiment_data: CompositeExperimentData, **options):
+    def _run_analysis(self, experiment_data: ExperimentData, **options):
         """Run analysis on circuit data.
```

Comment:
```suggestion
        """Run analysis on circuit data.
        
        This method dynamically initializes experiment data for each component experiment, 
        and add them to the child of the given ``expeirment_data``. Experiment result data 
        provided by the ``expeirment_data`` is marginalized as needed basis, and
        analysis class tied to the component experiment is called with the marginalized
        experiment results.        
```
Can you add more description how it works? It took long time for me to understand behavior of this method correctly. Especially I was confused by difference of `experiment_data.data()` and `experiment_data.child_data(i).data()` (I was thinking actual experiment result is in child but actually not, child will not be created until this method is called).

Reply:
Adding documentation here will not be seen by users since its an internal method, but I can add some more description to the class doc string